cg/apps/loqus.py

```diff
@@ -31,6 +30,7 @@ def load(
         family_ped_path: Optional[Path] = None,
         window_size: Optional[int] = None,
         gq_threshold: Optional[int] = None,
+        qual_gq: Optional[bool] = False,
```

Comment:
Could this be given a more descriptive name? Feel free to disregard, as someone who has not seen this code before I have no clue what `qual_gq` means 😅

Reply:
Oh, might be out of scope as well

Reply:
It's not so straightforward, I agree, but I think it's better to follow the naming we have in Loqusdb to not make it even more complex to follow